test/Github/Tests/Api/IntegrationTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Github\Tests\Api;
+
+class IntegrationTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function shouldFindRepositoriesForIntegration()
+    {
+        $api = $this->getApiMock();
+        $api->expects($this->once())
+            ->method('get')
+            ->with('/integration/installations');
+
+        $api->findInstallations();
+    }
+
+    /**
+     * @test
+     */
+    public function shouldGetRepositoriesFromInstallation()
+    {
+        $api = $this->getApiMock();
+        $api->expects($this->once())
+            ->method('get')
+            ->with('/installation/repositories');
+
+        $api->listRepositories('1234');
```

Comment:
`$userId`is required so this test should do. But can you give me some example or info on how to test the parameters content? I've thought about this test but I couldn't find on how to retrieve the passed parameters and check if `$params['user_id']` was equal to the passed value